### DISCUSSION TITLE
apollo-config加密后，java sdk  apollo-client调用报401

### DISCUSSION THREAD

**Discussion by bufx** [apolloconfig/apollo#4794] [Q&A]
目前，部署方案是：apollo-configservice 把内置的eureka启用了鉴权，即：开启了spring security

但是在第三方使用 apollo-client sdk调用apollo服务时报鉴权失败 401错误，请问apollo-client如何处理这种鉴权，在哪里配置用户名和密码

**Comment by nobodyiam** [comment]
You may enable the eureka auth via [3.2.9 apollo.eureka.server.security.enabled](https://www.apolloconfig.com/#/zh/deployment/distributed-deployment-guide?id=_329-apolloeurekaserversecurityenabled-%e9%85%8d%e7%bd%ae%e6%98%af%e5%90%a6%e5%bc%80%e5%90%afeureka-server%e7%9a%84%e7%99%bb%e5%bd%95%e8%ae%a4%e8%af%81)
